projects/etk-eof-implementation.md

```diff
@@ -0,0 +1,94 @@
+
+### Ethereum Improvement Proposals (EIPs) Summary:
+
+1. **[EIP-3540](https://eips.ethereum.org/EIPS/eip-3540)**: **EOF - EVM Object Format v1**
+   - Introduces a new contract code format for the Ethereum Virtual Machine (EVM) called EVM Object Format (EOF).
+
+2. **[EIP-3670](https://eips.ethereum.org/EIPS/eip-3670)**: **EOF - Code Validation**
+   - Focuses on pre-execution code validation.
+
+3. **[EIP-4200](https://eips.ethereum.org/EIPS/eip-4200)**: **EOF - Static relative jumps**
+   - Proposes static relative jumps as a new mechanism for code execution.
+
+4. **[EIP-4750](https://eips.ethereum.org/EIPS/eip-4750)**: **EOF - Functions**
+   - Introduces the concept of functions within the EOF format.
+
+5. **[EIP-5450](https://eips.ethereum.org/EIPS/eip-5450)**: **EOF - Stack Validation**
+   - Addresses stack validation during code execution.
+
+6. **[EIP-6206](https://eips.ethereum.org/EIPS/eip-6206)**: **EOF - JUMPF instruction**
+   - Introduces a new instruction for conditional jumps.
+
+7. **[EIP-7480](https://github.com/ethereum/EIPs/pull/7480)**: **EOF - Data section access instructions**
+   - Proposes instructions for accessing the data section within the EOF format.
+
+8. **[EIP-663](https://eips.ethereum.org/EIPS/eip-663)**: **Unlimited SWAP and DUP instructions**
+   - Proposes the removal of limitations on SWAP and DUP instructions.
+
+9. **[EIP-7069](https://eips.ethereum.org/EIPS/eip-7069)**: **Revamped CALL instructions**
+   - Proposes a revision to the CALL instructions.
+
+The document also mentions other proposed changes that do not yet have an assigned EIP, such as contract creation and restrictions on code and gas introspection.
+
+## Roadmap
+
+The first week or week and a half will be devoted to re-reading each of the EIPs that make up the EOF specification and putting together a technical document that specifies the necessary changes, particularities of each EIP and possible problems that may arise in their respective implementations.
+
+Regarding the implementation per se, while it is difficult to predict how long the implementations of each of the EIPs will take (some are trivial and others require a significant amount of work), I estimate that on average, about one week per EIP will be needed (including implementation, extensive individual testing and integration testing). At least in the case of ETK, which is an assembly language and its structure is relatively close to the final program bytecode.
+
+## Possible challenges
+
+- The EOF specification is still in draft status, so it is possible that some changes will be made to it during the development of the project. 
+- A bug in a compiler can be a big problem (and it is even worse when the deployed code is immutable). Extensive testing is needed.
+- The EOF specification is not yet implemented in any language, so there are no examples to take as a reference.
```

Comment:
not even partial? @lightclient was working on some concept implementation I think and maybe JS client too. Not 100% sure about it but I think there was some work done on this before Shapella

Reply:
There is some work here and there. For example, the Solidity repository has a branch with some progress, but it is not updated to the latest specification. My point was that, as the spec moves forward day by day, there is not yet a 100% finalized reference implementation. Anyway, maybe that was too strong a statement, I will rewrite it to make it clearer :)